src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Meta/WorkplaceMetadata.g.cs

```diff
@@ -43,6 +43,25 @@ public static Nox.Types.Text CreateName(System.String value)
             => Nox.Types.Text.From(value, NameTypeOptions);
         
     
+        /// <summary>
+        /// Type options for property 'ReferenceNumber'
+        /// </summary>
+        public static Nox.Types.ReferenceNumberTypeOptions ReferenceNumberTypeOptions {get; private set;} = new ()
+        {
+            StartsAt = 10,
+            IncrementsBy = 5,
+            Prefix = "WP-",
+            SuffixCheckSumDigit = true,
+        };
+    
+    
+        /// <summary>
+        /// Factory for property 'ReferenceNumber'
+        /// </summary>
+        public static Nox.Types.ReferenceNumber CreateReferenceNumber(System.String value)
```

Comment:
I think this method is useless because it would throw an exception anyways, but nothing to do with this PR, just mentioning maybe we can remove them for the types like ReferenceNumber.

Reply:
I guess we can remove all that are not creatable